standard/navbar.lua

```diff
@@ -1,122 +1,156 @@
-local p = {}
+-- Original source copied from Wikipedia on 2017-12-20.
+-- Modified (code style + using Class Module) on 2021-12-?? for liquipedia
 
-local getArgs
+local Class = require('Module:Class')
+local Logic = require('Module:Logic')
```

Comment:
Missing git header

Reply:
intended due to not sure if we want to add it to the git (the orig source is from wikipedia, i just heavily modified it)

Reply:
I think if it's in the git, it should have the git release pipeline. If it should be on git is the question then.
Does the license allow it (I think it does)?

Reply:
thats to be discussed
and once decided we can either close the base PR or add the headers
i mostly want this PR and the other one for clean code and so that it is easier to see the changes (so that we actually can decide if it is okay to add)